pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/AccessorMethodGenerationRule.java

```diff
@@ -48,31 +47,31 @@ public Object visit(ASTFieldAccess node, Object data) {
 
     @Override
     public Object visit(ASTVariableAccess node, Object data) {
-        JVariableSymbol sym = node.getReferencedSym();
-        if (sym instanceof JFieldSymbol) {
```

Comment:
The `if` here is kind of obsolete IMHO and could/should be inlined to resolve the overhead/boilerplate. This is only noise and takes away from the actual business requirement. 

What's your opinion on this?

Reply:
@oowekyala these two if statements are actually one. This is an unnecessary overhead I was trying to resolve.